comp/dogstatsd/pidmap/mock.go

```diff
@@ -0,0 +1,41 @@
+package pidmap
```

Comment:
Looks like the mock and the impl are essentially the same thing. 
I would either stub out these methods - or just provide the core implementation in place of the mock if there is little value im mocking out this component